packages/terra-form/src/Textarea.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import 'terra-base/lib/baseStyles';
+
+import './Textarea.scss';
+
+const propTypes = {
```

Comment:
Would it be better add the `rows` and `cols` props to the component or leave them for consumers to pass in as additional props ?